staging/src/k8s.io/apiserver/pkg/storage/storagebackend/factory/etcd3.go

```diff
@@ -134,6 +136,8 @@ func newETCD3Client(c storagebackend.TransportConfig) (*clientv3.Client, error)
 		grpc.WithBlock(), // block until the underlying connection is up
 		grpc.WithUnaryInterceptor(grpcprom.UnaryClientInterceptor),
 		grpc.WithStreamInterceptor(grpcprom.StreamClientInterceptor),
+		grpc.WithUnaryInterceptor(otelgrpc.UnaryClientInterceptor(global.Tracer(""))),
```

Comment:
just curious, why the empty string?

Reply:
empty string implies the default tracer.  In future versions of the otelgrpc plugin, the argument is removed entirely, and it only uses the default.

Reply:
same comment about being inert if the feature is off

Reply:
I removed the etcd integration for this PR, but will keep that in-mind when we add this in later.